estate/models/estate_property.py

```diff
@@ -0,0 +1,35 @@
+from odoo import models, fields
+from datetime import datetime, timedelta
+
+class EstateProperty(models.Model):
```

Comment:
We usually add 2 empty line defining class
```suggestion


class EstateProperty(models.Model):
```